src/QsCompiler/QirGeneration/Context.cs

```diff
@@ -51,7 +51,7 @@ static GenerationContext()
 
         /// <summary>
         /// The module used for QIR generation.
-        /// Generated functions to facilitate interoperability are created in a separate <see cref="InteropModule"/>.
+        /// Generated functions to facilitate interoperability are created via <see cref="Interop"/>.
```

Comment:
@bettinaheim, is this the best correction? I see that `Context` itself also appears to expose methods to facilitate interop.

Reply:
Yes, that sounds good. Thanks for finding it.